packages/api/lib/reconciliationReport.js

```diff
@@ -58,21 +66,35 @@ function convertToESCollectionSearchParams(params) {
 }
 
 /**
- *
- * @param {Object} params - request params to convert to database params
- * @returns {Object} object of desired parameters formated for database collection search
+ * convertToDBCollectionSearchObject      - Creates Postgres search object from
+ *                                          InternalRecReport Parameters
+ * @param {Object} params                 - request params to convert to database params
+ * @param {[Object]} params.collectionIds - List containing single Collection object
+ *                                          multiple or no collections will result in a
+ *                                          search object without a collection object
+ * @param {moment} params.endTimestamp    - ending report datetime ISO Timestamp
+ * @param {moment} params.startTimestamp  - beginning report datetime ISO timestamp
+ * @returns {[Object]}                    - array of objects of desired
+ *                                          parameters formatted for database collection
+ *                                          search
  */
-function convertToDBCollectionSearchParams(params) {
+function convertToDBCollectionSearchObject(params) {
   const { collectionIds, startTimestamp, endTimestamp } = params;
   // doesn't support search with multiple collections
-  const collection = collectionIds && collectionIds.length === 1
-    ? deconstructCollectionId(collectionIds[0]) : {};
-  const searchParams = {
-    updatedAt__from: dateToValue(startTimestamp),
-    updatedAt__to: dateToValue(endTimestamp),
-    ...collection,
-  };
-  return removeNilProperties(searchParams);
+  let collection = {};
+  if (collectionIds && collectionIds.length === 1) {
+    collection = deconstructCollectionId(collectionIds[0]);
+  } else {
+    log.info(`Multiple or no collections passed to convertToDBCollectionSearchObject ${JSON.stringify(params)}`);
```

Comment:
Seems kind of weird to just silently only use the first collection instead of throwing an error, but I realize that was/is the existing logic. Also though, is there any reason we can't support searching on multiple collection IDs in PG?

Reply:
I agree, and have concerns more generally about the reconciliation report logic, however for size/complexity reasons related to this PR, unless it was egregiously broken or I'm trying to avoid implementing logic refactors of that sort.   


>  only use the first collection instead of throwing an error, 

That said, that's not exactly what this is doing - it silently uses the first one if and only if you provide an array of length 1.  

Otherwise, it just silently *doesn't use a collection constraint at all*.

Reply:
Agreed that it is out of scope here